rust/src/pet/message/round.rs

```diff
@@ -0,0 +1,142 @@
+#![allow(dead_code)] // temporary
+
+use std::ops::Range;
+
+use sodiumoxide::crypto::{box_, sealedbox, sign};
+
+use super::{ROUND_TAG, TAG_RANGE};
+use crate::pet::PetError;
+
+// round box field ranges
+const ENCR_PK_RANGE: Range<usize> = 1..33; // 32 bytes
```

Comment:
Usually we try to align fields on 4 or 8 bytes boundaries. I don't know if it really has an impact here, but it would be interesting to benchmark it, and potentially add some padding.

Reply:
good idea, we should keep this in mind.